api/renter.go

```diff
@@ -5,18 +5,20 @@ import (
 	"time"
 
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/renter"
 	"github.com/NebulousLabs/Sia/types"
 )
 
 const (
-	duration   = 6000 // Duration that hosts will hold onto the file.
-	redundancy = 15   // Redundancy of files uploaded to the network.
+	duration     = 6000    // Duration that hosts will hold onto the file.
+	dataPieces   = 2       // Default data pieces per erasure-coded chunk
+	parityPieces = 10      // Default parity pieces per erasure-coded chunk
+	pieceSize    = 1 << 22 // Default size of one piece (4 MiB)
```

Comment:
Well, I think we want them to be settable at some point, which means they need to be passed as parameters to Upload.

I just realized that `duration` is redefined in `renter/negotiate.go`. Whoops. It should be passed in to `newHostUploader` as an argument.